cfpb/cfgov-refresh
# Upgrade Django to 1.11.27

## Changes

- Updates Django requirement to version 1.11.27

## Testing

1. Stop your local server, if it's running
1. Pull branch
1. `pip install -r requirements/local.txt` or `docker-compose build python2 python3`
1. Relaunch your local server
1. Visit http://localhost:8000/ or http://localhost:8333/ and click around to various places to confirm that everything is still working as expected

## Checklist
